docker-compose.yaml

```diff
@@ -2,7 +2,6 @@ version: '3'
 services:
   arangodb:
     image: arangodb/arangodb:3.4.3
-    container_name: arangodb
```

Comment:
Why did you remove this? It makes the name much longer and you changed the name of the test one

Reply:
Including the container name overrides the use of the `-p` flag. Even when I supplied a name for the testing container, for some reason that I don't fully get, starting that container would kill the dev container. In the end, not setting the container name this way and letting the system autogenerate names based on `-p` was the cleanest way to go.

Reply:
Ok, works for me